lib/filutils.sav.rb

```diff
@@ -0,0 +1,2528 @@
+# frozen_string_literal: true
```

Comment:
I think this file was committed on accident?

Reply:
Removed.